corehq/apps/tzmigration/timezonemigration.py

```diff
@@ -0,0 +1,190 @@
+import collections
+from copy import deepcopy
+import os
+
+from couchdbkit import ResourceNotFound
+from django.conf import settings
+from casexml.apps.case.cleanup import rebuild_case_from_actions
+from casexml.apps.case.models import CommCareCase, CommCareCaseAction
+
+from casexml.apps.case.xform import get_case_updates
+from corehq.apps.commtrack.processing import get_stock_actions
+from dimagi.utils.couch.database import iter_docs
+from corehq.apps.tzmigration import set_migration_started, \
+    set_migration_complete
+from corehq.apps.tzmigration.planning import PlanningDB
+from couchforms.dbaccessors import get_form_ids_by_type
+from couchforms import convert_xform_to_json
+from couchforms.models import XFormInstance
+from couchforms.util import adjust_datetimes, scrub_meta
+from corehq.apps.tzmigration import force_phone_timezones_should_be_processed
+
+
+def run_timezone_migration_for_domain(domain):
+    set_migration_started(domain)
+    _run_timezone_migration_for_domain(domain)
+    set_migration_complete(domain)
```

Comment:
I think at least until this is completely not manual, it's good to keep it around for a while. The db includes all the diffs, from which we can extract useful information. I think after the first few runs, I'll have a better idea of what exactly I want to be saving, and then we can dump that to a more permanent log file that we can refer to later in case someone complains about a change/inconsistency in their project data.